tests/unittests/test_google_sheets_supportsAllDrives_In_config.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from unittest import mock
+from tap_google_sheets import GoogleClient
+
+class Testsupports_all_drives(unittest.TestCase):
+    def test_supports_all_drives_not_in_config_file(self):
+        """To verify that when the supports_all_drives value is not given in config.json then set default value as False"""
+        # supports_all_drives not in config.json so 
+        supports_all_drives = None
+        client = GoogleClient('test', 'test', 'test', None, 'test', supports_all_drives)
+        self.assertEqual(client.supports_all_drives, False, "supports_all_drives got unexpected value")
+        
+    def test_supports_all_drives_other_than_true_in_config(self):
+        """To verify that when the supports_all_drives value is given other than true value in config.json then set default value False"""
+        
+        # provide supports_all_drives other then true value
+        supports_all_drives = 123
+        client =  GoogleClient('test', 'test', 'test', None, 'test', supports_all_drives)
+        self.assertEqual(client.supports_all_drives, False, "supports_all_drives got unexpected value")
+        
```

Comment:
We should not accept invalid values for the properties and fallback to default value. Can we throw an exception for these kind of scenarios?

Reply:
Added an exception for the invalid value of the supports_all_drives config parameter and also added a unit test for same.